### PR TITLE
Clmb/main: Updates readme and simplifies setup

### PR DESCRIPTION
**Pull request by User1m**
### Description

Updates readme and simplifies  setup

also includes in PR #103 that solved #86

### CODE REVIEW

**Review by mattbroussard — APPROVED** [review]
In the future, please use more descriptive commit messages than "updates" for local commits since we typically* use merge commits when merging PRs and so your local commits end up in the main history. I'm surprised the commitlint hook did not yell at you?

For this PR, it might make sense to select "squash and merge" when you merge it.

*I don't love this (in most other big repos I've worked in, the squash strategy was used) and I don't think it was ever really explicitly decided, but this is the pattern I've seen since I've been working in this repo -- it might be worth having a discussion with everyone actively developing in GOST and maybe setting a policy. There is a setting on Github that org admins can set to default the repo to a particular merge mode or even forbid the others.

File: README.md [hunk 1]
```diff
@@ -23,45 +23,130 @@ Each folder inside packages/ is considered a workspace. To see a list of all wor
...
-`npx knex seed:run`
+6. Run Server (Terminal 1)
 
 After that you should be able to serve the backend and frontend by running in both server and client folders.
 
-`yarn run serve`
+**_Ensure using node v14_**
+
+```
+> nvm use v14.19.0
```
[the remark this answers is not in the document]
Comment (Rapol) [reply README.md:116]: nvm doesnt do that automatically :( , you will need to add [this to your zshrc](https://github.com/nvm-sh/nvm#zsh).

File: README.md [hunk 2]
```diff
@@ -23,45 +23,130 @@ Each folder inside packages/ is considered a workspace. To see a list of all wor
...
+7. Visit `client_url/login` (e.g http://localhost:8080/#/login) and login w/ user set in Step 5.
+
+**NOTE:** if you only see a blank screen then ensure you've set the `packages/client/.env` up
+
+**NOTE:** if you get `Error: Invalid login: 534-5.7.9 Application-specific password required.` then you'll need to set an App Password (https://myaccount.google.com/u/0/apppasswords) replacing your `NODEMAILER_EMAIL_PW` with the new generated PW.
+
+![](./docs/img/gmail-app-password.png)
+
+
+8. When you get the login email link, change the redirected path from `localhost:3000/api/sessions/...` to your client_url e.g `localhost:8080/api/sessions/`
```
[the remark this answers is not in the document]
Comment (Rapol) [reply README.md:146]: The email login link should point to the URL of the localhost server. The local server should then redirect to the client using the `WEBSITE_DOMAIN` url

File: packages/server/package.json [hunk 3]
```diff
@@ -7,9 +7,14 @@
...
     "start": "node src",
+    "start:dev": "nodemon src",
+    "start:server": "npx knex migrate:latest && npx knex seed:run || true && yarn serve",
     "serve": "nodemon src",
-    "db:migrate": "knex migrate:latest",
-    "db:seed": "knex seed:run",
+    "db:migrate": "npx knex migrate:latest",
+    "db:rollback": "npx knex migrate:rollback",
+    "migrate:make": "npx knex migrate:make",
+    "env": "export $(cat .env)",
```
Comment (Rapol) [inline packages/server/package.json:16]: love this 👍 . Rolled your own dotenv

File: packages/server/src/index.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const { CronJob } = require('cron');
 const fs = require('fs').promises;
 const path = require('path');
+require('dotenv').config();
```
Comment (Rapol) [inline packages/server/src/index.js:5]: dotenv is called in ./configure.js in line 7 so I think we can remove it. Did you have a problem with env vars?

**Review by Rapol — COMMENTED** [review]
Looks good, thanks for this. I agree with most of the comments from matt. Just give them a quick review

**Comment by User1m** [comment]
> Looks good, thanks for this. I agree with most of the comments from matt. Just give them a quick review

Thansk @Rapol have updated

**Pull request merged**